docs/documentation/platform/pam/resources/aws-iam.mdx

```diff
@@ -0,0 +1,244 @@
+
+## How It Works
+
+Unlike database or SSH resources that require a Gateway for network connectivity, AWS Console access works differently. Infisical uses AWS STS (Security Token Service) to assume roles on your behalf and generates temporary federated sign-in URLs.
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant Infisical
+    participant PAM Role as PAM Role<br/>(Your AWS Account)
+    participant Target Role as Target Role<br/>(Your AWS Account)
+    participant Console as AWS Console
+
+    User->>Infisical: Request AWS Console access
+    Infisical->>PAM Role: AssumeRole (with ExternalId)
+    PAM Role-->>Infisical: Temporary credentials
+    Infisical->>Target Role: AssumeRole (role chaining)
+    Target Role-->>Infisical: Session credentials
+    Infisical->>Console: Generate federation URL
+    Console-->>Infisical: Signed console URL
+    Infisical-->>User: Return console URL
+    User->>Console: Open AWS Console (federated)
+```
+
+### Key Concepts
+
+1. **PAM Role**: An IAM role in your AWS account that trusts Infisical. This is the "bridge" role that Infisical assumes first.
+
+2. **Target Role**: The IAM role that end users will actually use in the AWS Console. The PAM Role assumes this role on behalf of the user.
+
+3. **Role Chaining**: Infisical uses AWS role chaining - it first assumes the PAM Role, then uses those credentials to assume the Target Role. This provides an additional layer of security and audit capability.
+
+4. **External ID**: A unique identifier (your Infisical Project ID) used in the trust policy to prevent [confused deputy attacks](https://docs.aws.amazon.com/IAM/latest/UserGuide/confused-deputy.html).
+
+## Prerequisites
+
+Before configuring AWS Console access in Infisical PAM, you need to set up two IAM roles in your AWS account:
+
+1. **PAM Role** - Trusted by Infisical, can assume target roles
+2. **Target Role(s)** - The actual roles users will use in the console
+
+<Info>
+  **No Gateway Required**: Unlike database or SSH resources, AWS Console access does not require an Infisical Gateway. Infisical communicates directly with AWS APIs.
+</Info>
+
+## Step 1: Create the PAM Resource
+
+The PAM Resource represents the connection between Infisical and your AWS account. It contains the PAM Role that Infisical will assume.
+
+### 1.1 Create the PAM Role Permissions Policy
+
+First, create an IAM policy that allows the PAM Role to assume your target roles. We recommend using a wildcard pattern so you can add new target roles without updating this policy.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Action": "sts:AssumeRole",
+    "Resource": "arn:aws:iam::<YOUR_ACCOUNT_ID>:role/<YOUR_PREFIX>-*"
+  }]
+}
+```
+
+<Tip>
+  Choose a naming convention for your target roles (e.g., `pam-*`, `infisical-*`, or `privileged-*`). This allows you to add new accounts without modifying the PAM Role's permissions.
```

Comment:
i think we should just role out with something like role/*

this prefix work adds more complexity